### PR TITLE
Update Linux install instructions

### PR DESCRIPTION
**Pull request by vojtechjelinek**
Update the Linux install instructions and remove some old unneeded instructions.

### CODE REVIEW

File: Installing-Oppia-(Linux;-Python-3).md [hunk 1]
```diff
@@ -143,26 +143,32 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
...
+   $ sudo apt install direnv
+   ```
+   
+6. Setup direnv into your shell. Add following lines to the end of `.bashrc` (see [here](https://askubuntu.com/a/127059) for where to find this file):
+   ```bash
+   eval "$(direnv hook bash)"
    ```
 
-   In the cloned `oppia` folder, run
+7. Create a virtual environment for oppia. Add file named `.envrc` into the parent folder of the oppia repository with this content:
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]: Can we provide instructions for how to create the virtual environment?

File: Installing-Oppia-(Linux;-Python-3).md [hunk 2]
```diff
@@ -143,26 +143,32 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
...
 
    ```console
-   pyenv local oppia
+   use python 3.7.10
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]: How does this specify that the virtual environment (and not just the global python 3.7.10 installation) should be used?

File: Installing-Oppia-(Linux;-Python-3).md [hunk 3]
```diff
@@ -143,26 +143,32 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
...
+   ```
+
+   Then ran this command in the same folder:
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]:
```suggestion
   Then run this command in the same folder:
```

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Installing-Oppia-(Linux;-Python-3).md (hunk 3 above)
Comment (vojtechjelinek) [reply Installing-Oppia-(Linux;-Python-3).md]: Done.

File: Installing-Oppia-(Linux;-Python-3).md (hunk 2 above)
Comment (vojtechjelinek) [reply Installing-Oppia-(Linux;-Python-3).md]: Ah, I forgot one step before this.

File: Installing-Oppia-(Linux;-Python-3).md (hunk 1 above)
Comment (vojtechjelinek) [reply Installing-Oppia-(Linux;-Python-3).md]: Done, I've rephrased the sentence.

**Review by U8NWXD — APPROVED** [review]

**Comment by vojtechjelinek** [comment]
@U8NWXD Will you update the instructions for Mac in a similar fashion?

**Pull request merged**